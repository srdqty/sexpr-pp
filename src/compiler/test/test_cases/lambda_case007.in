(lambda () 3 . 5)
