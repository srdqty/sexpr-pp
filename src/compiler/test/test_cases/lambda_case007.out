stdin:1:1: lambda: invalid syntax
